test/cpp/cpp_test_util.cpp

```diff
@@ -66,6 +90,16 @@ bool CloseValues(at::Tensor tensor1, at::Tensor tensor2, double rtol,
   return equal;
 }
 
+at::Tensor MakeRandomTensor(at::IntArrayRef size, at::ScalarType scalar_type) {
```

Comment:
Why?
We had one generator and we removed it since it was obfuscating the code.
Leave at::rand() alone, and if we really need and int-rand, add an helper. I don't think we need that.

Reply:
I had to do this because we have different random factories depending on the `scalar_type`. The tests iterate through all `scalar_type`s, so a factory with an unique signature was needed. What is your suggestion?

Reply:
I would just have us call at::rand() and at::randint() directly.
In the places where scalar type iteration is needed, and only there, we may have a wrapper if we really need to explore values up to max().

Reply:
Done.